packages/components/pie-modal/src/modal.scss

```diff
@@ -57,7 +96,54 @@
         background: #{dt.$color-overlay};
     }
 
-    & .c-modal-heading {
+    .c-modal-content {
+        // Modal content Custom Props
+        --modal-content-font-size: calc(var(--dt-font-size-16) * 1px);
+        --modal-content-font-weight: var(--dt-font-weight-regular);
+        --modal-content-line-height: calc(var(--dt-font-size-16-line-height) * 1px);
+        --modal-content-padding: var(--dt-spacing-e);
+        --modal-content-padding-block-start: var(--dt-spacing-a);
+
+        font-size: var(--modal-content-font-size);
+        line-height: var(--modal-content-line-height);
+        font-weight: var(--modal-content-font-weight);
+
+        padding-inline: var(--modal-content-padding);
+
+        overflow-y: auto;
+
+        &--scrollable {
+            background:
+                // Scroll shadow cover - a top-to-bottom opacity gradient from transparent to the component background colour
+                linear-gradient(to bottom, transparent, var(--dt-color-container-default) 75%) center bottom,
+                // Scroll shadow
+                linear-gradient(transparent, var(--dt-color-border-strong)) center bottom;
```

Comment:
The design tokens we currently have for elevation drop shadows are not compatible with methods to make the shadow fade away as you scroll. We'll have to work with design to try and find a longer-term solution for this.